.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+---
+defaults:
+  php_container: &php_container
+    docker:
+      - image: drupaldocker/php:7.1-cli
+    working_directory: /rpmbuild
+  rpmbuild_container: &rpmbuild_container
+    docker:
+      - image: quay.io/getpantheon/rpmbuild-fedora:22
+    working_directory: /rpmbuild
+  packagecloud_container: &packagecloud_container
+    docker:
+      - image: quay.io/getpantheon/packagecloud:latest
+    working_directory: /rpmbuild
+
+version: 2
```

Comment:
It may be more effort than desired, but v2.1 allows `commands` and `executors` to be defined to DRY up the config. It would replace many of the variable you've used.

For an example, take a look at https://github.com/pantheon-systems/pantheon_node/blob/master/.circleci/config.yml

Reply:
Yeah I have used those before. I just copied this from a similar rpmbuild repo in the interest of expedience. Since heirloom is going away, I probably won't expend effort refining the build model here.